drivers/bridge/bridge.go

```diff
@@ -218,14 +225,24 @@ func (c *networkConfiguration) fromLabels(labels map[string]string) error {
 			if c.EnableICC, err = strconv.ParseBool(value); err != nil {
 				return parseErr(label, value, err.Error())
 			}
-		case DefaultBridge:
-			if c.DefaultBridge, err = strconv.ParseBool(value); err != nil {
-				return parseErr(label, value, err.Error())
-			}
 		case DefaultBindingIP:
 			if c.DefaultBindingIP = net.ParseIP(value); c.DefaultBindingIP == nil {
 				return parseErr(label, value, "nil ip")
 			}
+		case DefaultBridgeLegacyIPv6Network:
```

Comment:
With this option, the IPAM doesn't even know about this range. What happens if the same range is used by some other network and IPAM gives out the same IP address as what was generated here using MAC address. 

Would it be better to allocate this generated IP address from IPAM during the `SetIPAddress` call?